lib/ansible/modules/network/ios/ios_l2_interfaces.py

```diff
@@ -0,0 +1,357 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for ios_l2_interfaces
+"""
+
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+DOCUMENTATION = """
+---
+module: ios_l2_interfaces
+version_added: 2.9
+short_description: Manage Layer-2 interface on Cisco IOS devices.
+description: This module provides declarative management of Layer-2 interface on Cisco IOS devices.
+author: Sumit Jaiswal (@justjais)
+notes:
+- Tested against Cisco IOSv Version 15.2 on VIRL
```

Comment:
The YAML indentation looks wrong here:
```
notes:
- Tested against Cisco IOSv Version 15.2 on VIRL
```
Shouldn't all list items be indented two spaces from the header like this?
```
notes:
  - note 1
  - note 2
```
Looks like this formatting is used throughout.

Reply:
updated